AirCastingTests/Logic/SessionCartTests/SessionCartViewModelTest.swift

```diff
@@ -0,0 +1,9 @@
+// Created by Lunar on 29/07/2021.
+//
+
+import XCTest
+@testable import AirCasting
+
+class SessionCartViewModelTest: XCTestCase {
```

Comment:
That's a start ;)

Reply:
And I think for now and only, because I've tried and lose. Will try again in near feature if allowed?